controllers/provisioning_controller.go

```diff
@@ -603,14 +599,6 @@ func (r *ProvisioningReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	// Check the Platform Type to determine the state of the CO
 	enabled := IsEnabled(r.EnabledFeatures)
-	if !enabled {
-		//Set ClusterOperator status to disabled=true, available=true
-		err = r.updateCOStatus(ReasonUnsupported, "Nothing to do on this Platform", "")
-		if err != nil {
-			return fmt.Errorf("unable to put %q ClusterOperator in Disabled state: %w", clusterOperatorName, err)
-		}
-		return nil
-	}
```

Comment:
The `Reconcile()` method will handle this case. 
@deads2k  Note that commenting this out means that the controller will be started, but it will always reconcile correctly. The `deadman` controller currently has a special case to ignore `ClusterOperators` with `condition[@type=Disabled]==true`. Removing this here makes that special case superfluous for the baremetal operator.